packages/react-wrapper/tests/demo/index.html

```diff
@@ -6,6 +6,6 @@
   </head>
   <body>
     <div id="app"></div>
-    <script lang="ts" src="demo.tsx"></script>
+    <script type="module" lang="ts" src="demo.tsx"></script>
```

Comment:
After update parcel, this tag is needed.

Reply:
👌🏻

Reply:
👌🏻